### ISSUE TITLE
Error running JavaScript message when launching OrcaSlicer

### ISSUE THREAD

**Issue by ChaosBlades** [SoftFever/OrcaSlicer#219]
I also tried the latest Artifact.
![image](https://user-images.githubusercontent.com/7530545/214067405-005eec2f-df22-4daa-9eec-a22e9b9071ba.png)

Also seeing a lot of Critical/Warning messages.
https://gist.github.com/ChaosBlades/ee8bbc64ca88a61debdbcd80d2358fdd

I'm running Pop!_OS 22.04 LTS x86_64

This build I found on Discord appears to be working without issue.
https://transfer.sh/xMF5nk/BambuStudio_ubu64.AppImage
![image](https://user-images.githubusercontent.com/7530545/214070336-631d75fe-85a0-40ae-a6c5-6230c8d11c12.png)

**Comment by ChaosBlades** [comment]
Issue was mentioned when the appimage builder was merged.
https://github.com/SoftFever/OrcaSlicer/pull/86

**Comment by ckvsoft** [comment]
Issue persists also on beta OrcaSlicer v1.6.2
Ubuntu

**Comment by xpsKING** [comment]
Same issue here on Nobara 37 (Fedora). Also turns the entire / drive read only until restart

**Comment by ckvsoft** [comment]
Issue persists also on non beta OrcaSlicer v1.6.2
Ubuntu

**Comment by Poikilos** [comment]
Confirmed in 1.6.3 final on Fedora 38

**Comment by ChaosBlades** [comment]
I also still get it on the same version but on pop os.

**Comment by byteofsoren** [comment]
Confirmed on ArchLinux.

```
10:48:21 PM: Error running JavaScript: Unsupported result type
10:48:21 PM: Error running JavaScript: Unsupported result type
```

**Comment by Flavius-Geani** [comment]
Confirmed on ArchLinux and gtk seems to be mad too, not sure if there is a relation.
`(orca-slicer-bin:183254): Gtk-CRITICAL **: 16:57:16.163: gtk_window_resize: assertion 'height > 0' failed

(orca-slicer-bin:183254): Gtk-CRITICAL **: 16:57:16.168: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed

..........
`

**Comment by rajinder-yadav** [comment]
## Version: V1.6.3

## System Info

Operating System: openSUSE Tumbleweed 20230628
KDE Plasma Version: 5.27.6
KDE Frameworks Version: 5.107.0
Qt Version: 5.15.10
Kernel Version: 6.3.9-1-default (64-bit)
Graphics Platform: X11
Processors: 24 × AMD Ryzen 9 5900X 12-Core Processor
Memory: 62.7 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3080/PCIe/SSE2
Manufacturer: Micro-Star International Co., Ltd.
Product Name: MS-7A38
System Version: 8.0

## JavaScript error message

![snap-20230630-02](https://github.com/SoftFever/OrcaSlicer/assets/439226/2eb57153-57ec-41e1-9e05-a79d8f0709cf)

## Terminal output

```sh
$ ./OrcaSlicer_ubu64.AppImage 
[2023-06-30 11:55:03.625702] [0x00007f43de37ca40] [trace]   Initializing StaticPrintConfigs
add font of HarmonyOS_Sans_SC_Bold returns 1
add font of HarmonyOS_Sans_SC_Regular returns 1
loading network module at /home/yadav/.config/OrcaSlicer/plugins/libbambu_networking.so
error, dlerror is /home/yadav/.config/OrcaSlicer/plugins/libbambu_networking.so: cannot open shared object file: No such file or directory
after dlopen, network_module is (nil)

(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.368: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.378: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.378: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.379: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.379: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.380: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.380: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.380: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.381: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.385: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.385: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.386: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.386: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.416: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.418: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.418: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.419: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.426: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.426: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.426: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.494: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.499: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.499: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.506: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.506: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.512: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.512: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.517: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.517: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.523: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.523: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.529: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.529: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.705: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.705: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
** (orca-slicer:16165): CRITICAL **: 11:55:08.706: Cannot register URI scheme wxfs more than once
** (orca-slicer:16165): CRITICAL **: 11:55:08.706: Cannot register URI scheme memory more than once
** (orca-slicer:16165): CRITICAL **: 11:55:08.708: Cannot register URI scheme wxfs more than once
** (orca-slicer:16165): CRITICAL **: 11:55:08.708: Cannot register URI scheme memory more than once
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.722: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.723: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.740: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.740: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.740: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.740: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.740: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.740: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.741: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.741: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.741: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.741: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.742: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.742: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.743: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.743: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.744: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): GLib-GObject-CRITICAL **: 11:55:08.744: invalid cast from 'wxPizza' to 'GtkCellLayout'
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.744: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.815: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.815: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.815: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.824: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.824: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.824: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.827: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.827: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.827: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.834: gtk_window_resize: assertion 'height > 0' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.835: gtk_widget_set_size_request: assertion 'width >= -1' failed
(orca-slicer:16165): Gtk-CRITICAL **: 11:55:08.835: gtk_widget_set_size_request: assertion 'width >= -1' failed
```

**Comment by nettucu** [comment]
Confirmed on Arch Linux, orca slicer 1.6.3;

Error running Javascript: Unsupported result type

**Comment by psi-4ward** [comment]
Several month later and still using PrusaSlicer :\

**Comment by GauthamYerroju** [comment]
Confirmed on Nobara 37, Gnome 43.2, X11 session.

Error running Javascript: Unsupported result type

**Comment by kylek29** [comment]
> Several month later and still using PrusaSlicer :\

You can't just "okay" the error and use it? That's how I use it on Ubuntu. It's a nuisance, but I haven't found anything it actually breaks.

**Comment by srsam89** [comment]
> > Several month later and still using PrusaSlicer :\
> 
> You can't just "okay" the error and use it? That's how I use it on Ubuntu. It's a nuisance, but I haven't found anything it actually breaks.

Not for me. I click ok, it opens the error in another window and then the application freezes for a minute before crashing.

**Comment by xpsKING** [comment]
I’ve just given up on a fedora build. BambuStudio has merged in most of
orca’s changes and they have a fedora app image

**Comment by mplogas** [comment]
Bambulabs latest release (Fedora build 1.07.02.51) has this JavaScript error fixed for me. OrcaSlicer 1.6.4b2 still shows the JS type error.

Manjaro Linux, KDE/Plasma

**Comment by discip** [comment]
I am getting the exact same error here on Manjaro Gnome with the latest release **[1.6.6](https://github.com/SoftFever/OrcaSlicer/releases/tag/v1.6.6)**.

**Comment by WZ9V** [comment]
Still getting this Orca 1.6.6 on Linux Mint 21.2

**Comment by iroCheese** [comment]
Same problem on Arch Linux with orca-slicer-bin-1.6.4-1.

**Comment by ChaosBlades** [comment]
Fixed or at least hidden on OrcaSlicer V1.7.0 Beta Release
